plugins/disassemble/disassemble_main.ml

```diff
@@ -353,7 +353,12 @@ let print_missing =
           that do not have semantics." "print-missing"
 
 let input = Extension.Command.argument
-    ~doc:"The input file" Extension.Type.("FILE" %: string =? "a.out" )
+    ~doc:"The input file" Extension.Type.("FILE" %: string =? "a.out")
+
+let link = Extension.Command.parameter
+    ~doc:"The input libraries to link with"
+    Extension.Type.("FILES" %: list string)
+    "link"
```

Comment:
Let's pick the `--libraries` as the default name for the parameter, with an alias `-l`. Also, let's turn it into `parameters` (plural) so that we can repeat the option in a conventional manner. I would prefer to safe the `link` name for a possible `link` plugin (I am envisioning that at some point of time we will move stub resolver there).